### ISSUE TITLE
Could not load configuration file that uses suppressions by message

### ISSUE THREAD

**Issue by lqc** [jshiell/checkstyle-idea#381]
Recently (8.6), Checkstyle added a feature to allow suppressions by message: https://github.com/checkstyle/checkstyle/issues/2804 

When I try loading a configuration that uses this feature, I get:

```
Unable to parse .../checkstyle-suppressions.xml - missing checks and id attribute
```

There is also an updated DTD in Checkstyle.

My guess is that the base version of Checkstyle needs to be updated, but I'm not sure how the backwards compatibility stuff works.

**Comment by jshiell** [comment]
Thanks for the report. We offer newer versions of Checkstyle, so that should be a problem (just select them in preferences) - however the DTDs are probably not available in the internal resolver. I'll add them anon, as it appears Checkstyle's website & hence docs are currently down for Sourgeforge maintenance.

Do you have any test files to show this case/

**Comment by jshiell** [comment]
We were indeed missing a few DTDs. I've added them - if you can please supply a test file then I'll check this wasn't the only problem, and then get it released. Thank you.

**Comment by jshiell** [comment]
Released in 5.18.1. Please reopen if you have any problems.
